dom.go

```diff
@@ -142,9 +191,11 @@ func (h *HTML) restoreHTML(prev *HTML) {
 			h.node.Set(name, value)
 		}
 	}
-	for name := range prev.properties {
-		if _, ok := h.properties[name]; !ok {
-			h.node.Set(name, nil)
+	if !h.new {
+		for name := range prev.properties {
```

Comment:
`prev` may be a populated, but incompatible tag.  However `h.new` will be valuable for subsequent PRs (comments to follow in the issue body).